### DISCUSSION TITLE
Teensy++ 2 Potentiometers values unstable

### DISCUSSION THREAD

**Discussion by ArrivalH** [shanteacontrols/OpenDeck#180] [Q&A]
Hi, i've built a simple setup using all 8 analogue inputs on the teensy board along with all the digital buttons and 1 encoder and i'm have issues where the pots are giving unstable data. It sporadically jumps between two close values but is fine when it is either at 1 or 127. I've tried a different pot to see if it was a hardware issue but the problem remains. it happens across all the analogue inputs and i'm wondering if a resistor inline somewhere might help.
Its running opendeck firmware 5(something) I am also unable to get the board to appear in the browser when in bootloader mode to update the firmware.
any suggestions would be really appreciated. cheers

**Comment by paradajz** [comment]
Hi, I would suggest to reflash the board with the latest firmware, which is v7.4.0. Much work has been done on stabilizing analog values in the mean time.

**Reply by ArrivalH** [reply]
Thank you very much for your help! and thanks for all your work! 
I'm struggling to reflash the firmware, i can't find the OpenDeck/src directory to build the firmware.
You can tell i'm no programmer, I mainly just like to dub.
thanks

<img width="898" alt="Screenshot 2022-11-16 at 15 59 45" src="https://user-images.githubusercontent.com/118309736/202231572-4cc098ad-5a63-47d0-8fc5-81f392f9f060.png">

**Comment by paradajz** [comment]
Well, how did you flash it the first time?

**Reply by ArrivalH** [reply]
Yea good response, It was a few years ago and i can't remember.
I'll keep trying

**Reply by paradajz** [reply]
All the instructions are here: https://github.com/shanteacontrols/OpenDeck/wiki/Flashing-AVR-boards

**Reply by ArrivalH** [reply]
do i have to use a arduino to flash it or is it possible from terminal alone? thanks
